lib/connextclient/ConnextClient.ts

```diff
@@ -332,6 +334,35 @@ class ConnextClient extends SwapClient {
         channelBalancePromises.push(this.channelBalance(currency));
       }
       await Promise.all(channelBalancePromises);
+
+      for (const [currency, address] of this.tokenAddresses) {
+        const remoteBalance = this.inboundAmounts.get(currency) || 0;
```

Comment:
Could we extract this one into a function that takes `remoteBalance`, `localBalance`, `totalBalance`, `alertThreshold`, `emit` as arguments?

This way we can test the business logic as a pure function.